network/server/tchannelthrift/node/service.go

```diff
@@ -300,6 +302,13 @@ func (s *service) FetchBatchRaw(tctx thrift.Context, req *rpc.FetchBatchRawReque
 	return result, nil
 }
 
+// stubb function
```

Comment:
Perhaps update this to be more standard, i.e. `// TODO(jskelcy): implement the stubbed functions`